registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/BankNameValidator.java

```diff
@@ -0,0 +1,13 @@
+package de.adorsys.registry.manager.service.validator.property;
+
+import de.adorsys.registry.manager.service.model.AspspBO;
+import de.adorsys.registry.manager.service.model.AspspValidationReportBO;
+
+public class BankNameValidator implements PropertyValidator {
+    private static final String EMPTY_PROPERTY_VALIDATION_ERROR_MESSAGE = "Bank name is empty";
+
```

Comment:
We have a constraint on the front that Banc Name should consist of word characters, whitespace characters, + german special symbols. I believe such a rule should be applied here as well.

Reply:
Yeah, why not, I'll do that, thanks